### PR TITLE
fix bug: if key not exist, will return default value instead of Variant

### PR DESCRIPTION
**Pull request by lidaobing**

### CODE REVIEW

**Comment by lidaobing** [comment]
on the PR of yesterday(#86), I make a mistake. when the config not in `IBus.Config`, it will return the default value, which cause crash in the following unwrap code.

in this patch, I fix this problem, and because `Option.wrap` and `Option.unwrap` only used once, so I inline this two function in `Option.read` and `Option.write`.

thanks

**Comment by lidaobing** [comment]
@yongsun @tchaikov

File: wrapper/ibus/setup/main.py [hunk 1]
```diff
@@ -72,12 +72,12 @@ class Option(object):
...
     """
     config = ibus.Bus().get_config()
-    wrappers = {
+    __wrappers = {
         type(True): glib.Variant.new_boolean,
         type(1): glib.Variant.new_int32,
         type('str'): glib.Variant.new_string,
     }
-    unwrappers = {
+    __unwrappers = {
```
Comment (tchaikov) [inline wrapper/ibus/setup/main.py]: how about something like:

```py
def __init__(self, name, default):
  self.name = name
  self.default = default
  self.__wrap = self.__wrappers[type(self.default)]
  self.__unwrap = self.__unwrappers[type(self.default)]

def read(self):
  section, key = self.__get_config_name()
  wrapped =  self.config.get_value(section, key)
  return self.default if wrapped is None else self.__unwrap(wrapped)
```

File: wrapper/ibus/setup/main.py [hunk 2]
```diff
@@ -89,17 +89,14 @@ def __init__(self, name, default):
...
+        res =  self.config.get_value(section, key)
+        if res is None:
+            return self.default
+        return self.__unwrappers[type(self.default)](res)
 
     def write(self, v):
         section, key = self.__get_config_name()
-        return self.config.set_value(section, key, self.wrap(v))
-
-    def wrap(self, v):
```
Comment (tchaikov) [inline wrapper/ibus/setup/main.py]: actually, i like this little helper apart from its name -- would be better if it was started with `__`. as it's "private".

**Comment by lidaobing** [comment]
@tchaikov your comments has been resolved.

**Comment by lidaobing** [comment]
@tchaikov now it's only one commit.

**Comment by lidaobing** [comment]
@tchaikov still has problem with `MultiMappingOption`, wait a minute

**Comment by tchaikov** [comment]
sure.

**Comment by lidaobing** [comment]
@tchaikov 1. now it can deal with `MappingOption`(which use `[string]`), 2. replace the unwrap func with builtin `unpack`

**Comment by tchaikov** [comment]
@lidaobing thanks for your contribution!

**Pull request merged**